collections/nemo_nlp/nemo_nlp/data/datasets/utils.py

```diff
@@ -936,6 +938,19 @@ def write_files(data, outfile):
             f.write(item)
 
 
+def calc_class_weights(label_freq):
+    # Goal is to give more weight to the classes with less samples
+    # so as to match the one with the higest frequency. We achieve this by
+    # dividing the highest frequency by the freq of each label.
+    # Example -
+    # [12, 5, 3] -> [12/12, 12/5, 12/3] -> [1, 2.4, 4]
+
+    most_common_label_freq = label_freq[0]
```

Comment:
either:
1. check that label_freq is sorted
2. update docstring to show that label_freq has to be sorted so that label_freq[0] is the most frequent element

also, use """ for docstring